### DISCUSSION TITLE
Awkard-ish notation of getbalance()

### DISCUSSION THREAD

**Discussion by Namekinek0** [1200wd/bitcoinlib#330] [Q&A] [answered]
Hi guys, thank you for this project!

I've sent 0.00006691 BTC to [this address](https://blockstream.info/testnet/address/tb1qwan3hcde76dvms76nmhf5h26mxvflcqnt5g4mn)

but when I run this code 

    from crypto import * #wallet from here
    from bitcoinlib.services.services import Service
    
    address = 'tb1qwan3hcde76dvms76nmhf5h26mxvflcqnt5g4mn'
    
    w.utxos_update()
    
    s = Service(network='testnet')
    
    check = s.getbalance(address)
    print(check)

I'd have guessed the output of getbalance() would have looked like what I see from clw, that is `0.00006691`
Instead, getbalance() gives me `6691`. Why the integer? Is there a way to get the result with decimals as in clw? Thank you.

**Comment by mccwdev** [comment, marked as answer]
The getbalance() method returns the value in Satoshi's, the smallest denominator for Bitcoin.

You can get the value in bitcoins by dividing by 100000000 or 10**8:
```python
>>> amount / 10**8 
0.00006691
```

Or you could use bitcoinlib Value class to conveniently display values

```python
>>> from bitcoinlib.values import Value
>>> Value.from_satoshi(6691, 1).str()
'0.00006691 BTC'

```

**Reply by Namekinek0** [reply]
Ahhh, that's why. All clear, bless you!
